bin/metrics-temperature.rb

```diff
@@ -58,7 +58,7 @@ def run
           key, value = line.split(':')
           key = key.downcase.gsub(/\s/, '')
 
-          if key.start_with?('temp', 'core', 'loc', 'power', 'physical')
+          if key.start_with?('temp', 'core', 'loc', 'power', 'physical', 'packageid')
```

Comment:
can you expand on what new CPU archs are now supported? Sorry I have not really worried about CPU temp in a while since I have all my stuff in aws.

Reply:
Sensors output for 6th and 7th generation Intel now displays overall CPU temperature as Package id instead of Physical id like 4th generation did.

Example:
Package id 0:  +37.0°C  (high = +80.0°C, crit = +100.0°C)
Core 0:        +37.0°C  (high = +80.0°C, crit = +100.0°C)
Core 1:        +35.0°C  (high = +80.0°C, crit = +100.0°C)
Core 2:        +37.0°C  (high = +80.0°C, crit = +100.0°C)
Core 3:        +37.0°C  (high = +80.0°C, crit = +100.0°C)